board/safety/safety_hyundai.h

```diff
@@ -49,16 +49,19 @@ static uint8_t hyundai_get_counter(const CANPacket_t *to_push) {
   uint8_t cnt = 0;
   if (addr == 0x260) {
     cnt = (GET_BYTE(to_push, 7) >> 4) & 0x3U;
-  } else if (addr == 0x386) {
+  } 
+  if (addr == 0x386) {
```

Comment:
it's important that these tools don't dictate our style. let's revert and revisit this